site/content/docs/manifest/backend-service.en.md

```diff
@@ -1,63 +1,142 @@
+              test:
+                deployment:
+                  rolling: "recreate"
+                count: 1
+        ```
+
+    === "Internal Application Load Balancer"
+
+        ```yaml
+        # Your service is reachable at:
+        # http://api.${COPILOT_ENVIRONMENT_NAME}.${COPILOT_APPLICATION_NAME}.internal
+        # behind an internal load balancer only within your VPC.
         name: api
         type: Backend Service
-
-        # Your service is reachable at "http://api.${COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080" but is not public.
-
-        # Configuration for your containers and service.
+    
         image:
           build: ./api/Dockerfile
           port: 8080
-          healthcheck:
-            command: ["CMD-SHELL", "curl -f http://localhost:8080 || exit 1"]
-            interval: 10s
-            retries: 2
-            timeout: 5s
-            start_period: 0s
-
-        cpu: 256
-        memory: 512
-        count: 1
-        exec: true
 
-        storage:
-          volumes:
-            myEFSVolume:
-              path: '/etc/mount1'
-              read_only: true
-              efs:
-                id: fs-12345678
-                root_dir: '/'
-                auth:
-                  iam: true
-                  access_point_id: fsap-12345678
+        http:
+          path: '/'
+          healthcheck:
+            path: '/_healthcheck'
+            success_codes: '200,301'
+            healthy_threshold: 3
+            interval: 15s
+            timeout: 10s
+            grace_period: 30s
+          deregistration_delay: 50s
 
         network:
           vpc:
             placement: 'private'
-            security_groups: ['sg-05d7cd12cceeb9a6e']
 
-        variables:
-          LOG_LEVEL: info
-        env_file: log.env
+        count:
+          range: 1-10
+          cpu_percentage: 70
+          requests: 10
+          response_time: 2s
+
         secrets:
-          GITHUB_TOKEN: GITHUB_TOKEN
-
-        # You can override any of the values defined above by environment.
-        environments:
-          test:
-            deployment:
-              rolling: "recreate"
-            count:
-              spot: 2
-          production:
-            count: 2
-    ```
+          GITHUB_WEBHOOK_SECRET: GH_WEBHOOK_SECRET
+          DB_PASSWORD:
+            secretsmanager: 'demo/test/mysql:password::'
+        ```
+
+    === "With a domain"
+
+        ```yaml
+        # Assuming your environment has private certificates imported, you can assign
+        # an HTTPS endpoint to your service.
+        # See https://aws.github.io/copilot-cli/docs/manifest/environment/#http-private-certificates
+        name: api
+        type: Backend Service
+    
+        image:
+          build: ./api/Dockerfile
+          port: 8080
+
+        http:
+          path: '/'
+          alias: 'v1.api.example.com'
+          hosted_zone: AN0THE9H05TED20NEID # Insert record for v1.api.example.com to the hosted zone.
```

Comment:
hahaha clever

Reply:
That wasn't me! This feels like a @huanjani thing to do